client/me/help/help-contact-form/index.jsx

```diff
@@ -99,6 +114,17 @@ export const HelpContactForm = React.createClass( {
 		}
 	},
 
+	doQandASearch() {
+		const query = this.state.subject + ' ' + this.state.message;
+		wpcom.getQandA( query, ( error, data ) => {
+			if ( null === error ) {
+				this.setState( { qanda: data } );
+			} else {
+				this.setState( { qanda: [] } );
+			}
+		} );
```

Comment:
if you leave out the callback you should be free to use the Promise convention, where `fn` will be `undefined` to wpcom

```js
wpcom.getQandA( query )
	.then( qanda => this.setState( { qanda } ) )
	.catch( () => this.setState( { qanda: [] } ) );
```